unit-tests/src/test/java/org/eclipse/collections/impl/block/function/Functions2Test.java

```diff
@@ -32,6 +37,54 @@ public void throwing()
                 }).value(null, null));
     }
 
+    @Test
+    public void throwingWithSuccessfulCompletion()
+    {
+        Assert.assertEquals("abcdef",
+                Functions2.throwing((ThrowingFunction2<String, String, String>)
+                        (argument1, argument2) -> argument1.concat(argument2)).value("abc", "def"));
+    }
+
+    @Test
+    public void min()
+    {
+        Function2<Integer, Integer, Integer> minFunction = Functions2.min(Comparators.naturalOrder());
+        Assert.assertEquals((Integer) 1, minFunction.value(5, 1));
```

Comment:
Use `Integer.valueOf(1)` here instead of casting. Same comment for other auto-boxing places below.

Reply:
Done.